ONSdigital/dp-census-search-prototypes
# Prototype - Load places with boundary file

### What

A go application to load documents containing places with boundary files that are searchable into an elasticsearch instance.

Also includes:

- Documentation for running go application and how to query elastic to find documents.
- Helpful list of commands
- General repo setup files

### How to review

- Follow documentation in README.md to run go app (script) and load data into an elasticsearch cluster (versions 6.7 or 6.8)
- Use queries in README.md to test your local index and the results are the same as described in documentation
- Check spelling/grammar (typos) etc. 

### Who can review

Anyone
